source/reference/limitations.txt

```diff
@@ -54,6 +54,10 @@ General Limitations
 - `Queryable Encryption
   <https://www.mongodb.com/docs/v6.0/core/queryable-encryption/>`__ is
   not supported.
+- After you replace the ``mongosync`` binary during an upgrade or a
+  downgrade, you should drop all non-system databases in the destination
+  cluster. Syncing operations will restart from the beginning when you
```

Comment:
Can we add "before starting the new binary" to this sentence? I think we should be clear that dropping must happen before the new binary is started.